### PR TITLE
Add delete button for revisions

### PR DESCRIPTION
**Pull request by RobinRSchulz**
Allow admins to delete job and agent revisions.

### CODE REVIEW

File: src/main/webapp/deleteRevisionAndTableEntry.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+//name is an optional parameter for job deletion history and system config history
+function removeEntryFromTable(id, timestamp, name) {
+    console.log("tedst");
+    var confirmPhrase = 'Do you really want to delete the history entry ' + timestamp + '?';
```
Comment (stefanbrausch) [inline src/main/webapp/deleteRevisionAndTableEntry.js]: Can we localize this?

File: src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryRootAction/history.jelly [hunk 2]
```diff
@@ -44,7 +46,7 @@
           <j:otherwise>
             <br/>
             <div>
-              <f:form method="post" action="diffFiles" name="diffFiles" enctype="multipart/form-data">
```
Comment (stefanbrausch) [inline src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryRootAction/history.jelly]: We don't need the name tag?
Maybe we need it for Selenium tests later?

File: src/main/java/hudson/plugins/jobConfigHistory/JobConfigHistoryStrategy.java [hunk 3]
```diff
@@ -27,8 +27,11 @@
 import hudson.ExtensionPoint;
 import hudson.model.Describable;
 import hudson.model.Descriptor;
+import hudson.model.Node;
 import jenkins.model.Jenkins;
 
+import java.io.File;
```
Comment (stefanbrausch) [inline src/main/java/hudson/plugins/jobConfigHistory/JobConfigHistoryStrategy.java]: Only imports without any other changes?

Comment (RobinRSchulz) [reply src/main/java/hudson/plugins/jobConfigHistory/JobConfigHistoryStrategy.java]: done. (removed it)

File: pom.xml [hunk 4]
```diff
@@ -114,7 +114,7 @@
     </developers>
 
     <properties>
-        <jenkins.version>2.60.3</jenkins.version>
+<!--        <jenkins.version>2.60.3</jenkins.version>-->
```
Comment (stefanbrausch) [inline pom.xml]: activate Jenkins version again

**Pull request merged**